charter/index.html

```diff
@@ -167,40 +172,41 @@ <h1 id="title">PROPOSED Solid Working Group Charter</h1>
 
       <div id="background" class="background">
         <h2>Motivation and Background</h2>
-        <p>It is common for web service providers to require users to surrender control over their own data, and store them in a way that is often restricted to certain systems, or can only be accessed by the applications that created it.</p>
-
-        <p>Solid is largely based on existing open web standards, and extends them to give priority to individual and community autonomy. It allows them to keep authority over their identity, data, and privacy, and gives them the freedom to select applications and services that suit their needs.</p>
-
-        <p>Solid defines the concept of storage (one feature of the construct informally referred to as Solid Pods), in which users place data and control access to it, and a suite of protocols for applications to use storages.</p>
-
-        <p>Solid offers several advantages over more traditional architectures for data use by web services today, including:</p>
+        <p>It is common for web service providers to require users to surrender control over their own data, and store them in a way that is often restricted to certain systems, or can only be accessed by the applications that created them.
+        </p>
 
-        <ul>
-          <li>Storage is controlled by individuals or organizations, and can be delegated to other parties.</li>
-          <li>Interoperability between applications and various storage server implementations of the protocol.</li>
-          <li>A decentralized trust model enabling authentication and authorization mechanisms across the web.</li>
-        </ul>
+        <p><span class=new>A number of initiatives, such as <a href="https://indieweb.org/">IndieWeb</a>, <a href="https://www.w3.org/community/unhosted/">Unhosted</a> and <a href="https://solidproject.org/">Solid</a>, have emerged to propose alternative models for web applications.
```

Comment:
Why highlight these two?  What is the user base, how many DAU?  This looks weak, imho.

Reply:
Feel free to suggest others if you think they are more relevant. Those were the ones that I was most aware of, and which I believe the Solid community has interacted with in the past.